### PR TITLE
Add the 'OT' prefix to all macros

### PR DESCRIPTION
**Pull request by LeZhang2016**

### CODE REVIEW

File: src/web/pskc-generator/pskc.cpp [hunk 1]
```diff
@@ -48,8 +48,8 @@ void Pskc::SetSalt(const uint8_t *aExtPanId, const char *aNetworkName)
     memcpy(mSalt, saltPrefix, strlen(saltPrefix));
     cur += strlen(saltPrefix);
 
-    memcpy(mSalt + cur, aExtPanId, EXTEND_PAN_ID_LEN);
-    cur += EXTEND_PAN_ID_LEN;
+    memcpy(mSalt + cur, aExtPanId, OT_EXTEND_PAN_ID_LEN);
```
Comment (jwhui) [inline src/web/pskc-generator/pskc.cpp]: nit: `OT_EXTENDED_PAN_ID_LEN`

File: src/web/wpan-controller/dbus_gateway.hpp [hunk 2]
```diff
@@ -34,7 +34,7 @@
 #ifndef DBUS_GATEWAY_HPP
 #define DBUS_GATEWAY_HPP
 
-#define INET6_ADDRSTRLEN 46
+#define OT_INET6_ADDRSTRLEN 46
```
Comment (jwhui) [inline src/web/wpan-controller/dbus_gateway.hpp]: nit: `OT_INET6_ADDR_STR_LEN`?

File: src/web/pskc-generator/pskc.hpp [hunk 3]
```diff
@@ -34,11 +34,11 @@
...
 #define PSKC_HPP
 
-#define EXTEND_PAN_ID_LEN 8
-#define PBKDF2_SALT_MAX_LEN 30
-#define PSKC_LENGTH 16
-#define ITERATION_COUNTS 16384
-#define MAX_PASSPHRASE_LEN 30
+#define OT_EXTEND_PAN_ID_LEN 8
+#define OT_ITERATION_COUNTS 16384
+#define OT_MAX_PASSPHRASE_LEN 30
```
Comment (jwhui) [inline src/web/pskc-generator/pskc.hpp]: nit: `OT_PASSPHRASE_MAX_LEN` to be consistent with `OT_PBKDF2_SALT_MAX_LEN`

File: src/web/web-service/web_service.cpp [hunk 4]
```diff
@@ -47,29 +47,29 @@
...
+#define OT_REQUEST_METHOD_POST "POST"
+#define OT_RESPONSE_SUCCESS_STATUS "HTTP/1.1 200 OK\r\n"
+#define OT_RESPONSE_HEADER_LENGTH "Content-Length: "
+#define OT_RESPONSE_HEADER_TYPE "Content-Type: application/json\r\n charset=utf-8"
+#define OT_RESPONSE_PLACEHOLD "\r\n\r\n"
+#define OT_RESPONSE_FAILURE_STATUS "HTTP/1.1 400 Bad Request\r\n"
+#define OT_SET_NETWORK_URL "^/settings$"
+
+#define OT_BORDER_ROUTER_PORT 49191
+#define OT_EXTENDED_PANID_LENGTH 8
```
Comment (jwhui) [inline src/web/web-service/web_service.cpp]: You use `LENGTH` here and `LEN` above, prefer consistency across.

File: src/web/web-service/web_service.cpp [hunk 5]
```diff
@@ -47,29 +47,29 @@
...
-#define RESPONSE_FAILURE_STATUS "HTTP/1.1 400 Bad Request\r\n"
-
-#define NETWORK_NAME_LENGTH 16
-#define EXTENDED_PANID_LENGTH 8
-#define HARDWARE_ADDRESS_LENGTH 8
-#define PANID_LENGTH 4
-#define BORDER_ROUTER_PORT 49191
-#define PUBLISH_SERVICE_INTERVAL 20
-#define PSKC_MAX_LENGTH 16
+#define OT_ADD_PREFIX_URL "^/add_prefix"
```
Comment (jwhui) [inline src/web/web-service/web_service.cpp]: These are paths, not URLs, right?

Comment (LeZhang2016) [inline src/web/web-service/web_service.cpp]: yes, these should be path. Thanks.

**Comment by LeZhang2016** [comment]
@jwhui , i have solved the conflict problem on this PR. Thanks.

**Pull request merged**